chapter4.Rmd

```diff
@@ -361,7 +361,7 @@ success_msg("Phew, it seems that R is gender-neutral. Or maybe it just wants to
 --- type:NormalExercise xp:100
 ## Ordered factors
 
-Since `"Male"` and `"Female"` are unordered (or nominal) factor levels, R returns a warning message, telling you that the greater than operator is not meaningful. As seen before, R attaches an equal value to the levels for such factors. 
+Since `"Male"` and `"Female"` are non-ordered (or nominal) factor levels, R returns a warning message, telling you that the greater than operator is not meaningful. As seen before, R attaches an equal value to the levels for such factors. 
```

Comment:
Keep the word "unordered" here. It's actually a technical term for a factor variable in R where order doesn't matter. For example, Male and Female are unordered, but Unsatisfied/Neutral/Satisfied would be ordered factors, etc.